codespell_lib/data/dictionary.txt

```diff
@@ -9065,9 +9065,18 @@ imigrated->emigrated, immigrated,
 imigration->emigration, immigration,
 imilar->similar
 iminent->eminent, imminent, immanent,
```

Comment:
I know it's not your change, but can we swap this so imminent is the first suggestion, given it's only one letter missing.

Reply:
0ce5399